dist/index.html

```diff
@@ -6,10 +6,95 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
+    <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="container-parent">
+      <div class="container-header">
+        <div class="container-logo">
+            <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" border="0" id="mainlogo">
+        </div>
+        <div class="container-navSection">
+          <div class="container-nav">
+                <div class="menu"> <a href="home.asp" class="fas fa-home">Home</a> </div>
```

Comment:
**Lines 19-22:**

- Over indented by 4 spaces.
- No need to add empty space between HTML tags. If its becoming an eye-sore, make a new line and indent child elements by two spaces.
- I see you’re having an issue formatting text inside an HTML element. Have you considered using a `<span></span>` tag to contain your text?

Reply:
resolved 4 space indentation

added some space in the container nav div to make it easier to see child elements

removed anchor tags and replaced them with < p > tags for now. wrapped font-awesome icons in span tag - the colors are changing upon hover on the div

Reply:
The `<span></span>` tag comment was meant to encourage you to do something like this:

```
<div class="menu"><a href="about.asp" class="fas fa-user"><span>About Me</span></a></div>
```

Or:

```
<div class="menu">
  <a href="about.asp" class="fas fa-user"><span>About Me</span></a>
</div>
```

With hope that the `<span>` would allow you to maintain the space between the words "About" and "Me".